database/v2/seeders/20191021130637-demo-user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const hashHelpers = require('../../../api/helpers/hashHelpers');
+
+async function createMockUser(i) {
+    const data = await hashHelpers.createHash("password");
+    const obj = {
+      userName: `John Doe`,
+      email: `demo${i}@demo.com`,
+      password: data,
+      role: 'User',
+      locale: 'ru',
+      resetPasswordToken: null,
+      createdAt: new Date(),
+      updatedAt: new Date()
```

Comment:
These fields have default value, you can skip them

Reply:
Done